engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/stores/ExecutionStore.scala

```diff
@@ -274,24 +281,24 @@ sealed abstract class ExecutionStore private[stores](statusStore: Map[JobKey, Ex
         // Even if the key is runnable, if it's a call key and there's already too many queued jobs,
         // don't start it and mark it as WaitingForQueueSpace
         // TODO maybe also limit the number of expression keys to run somehow ?
-        case callKey: CallKey if runnable && queuedJobsAboveThreshold =>
+        case callKey: CallKey if runnable && startableJobLimit <= 0 =>
```

Comment:
the code above keeps `startableJobLimit` from being less than zero
```suggestion
        case callKey: CallKey if runnable && startableJobLimit == 0 =>
```

Reply:
https://www.phrases.org.uk/meanings/belt-and-braces.html